src/screens/SignUp/index.tsx

```diff
@@ -36,16 +44,41 @@ function SignUp() {
     console.log(data);
   };
 
+  const changeLanguage = (lang: string): void => {
+    i18next.changeLanguage(lang);
+    setLanguage(lang);
+  };
+
   return (
     <div className="row full-height center middle">
       <form
         className={`column full-height center space-around ${styles.formContainer}`}
         onSubmit={handleSubmit(onSubmit)}
       >
-        <img className={styles.formImage} src={wLogo} alt="wolox-logo" />
+        <div className="full-width row center">
```

Comment:
I have a problem with feature because the idea with button for change language is use for all screens (pages) and components but, with this implementation only you can access to this functionality from register screen, please move this.